source/release-notes.txt

```diff
@@ -10,6 +10,30 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.44.0
+----------------
+
+*Released September 3, 2024*
+
+New Features:
+
+- Support for users working with data stored on different connections 
+  stored in the Compass connections window. (:issue:`COMPASS-6410`).
+- Adds per-connection proxy settings (:issue:`COMPASS-8142`).
+- Enables proxy support feature flag  (:issue:`COMPASS-8167`).
+
+Bug Fixes:
+
+- Stream import errors to the log file with proper back pressure 
+  (:issue:`COMPASS-7820`).
+- In the bulk update preview, convert array indexes from strings to 
+  numbers (:issue:`COMPASS-8218`).
+- Bump shell-bson-parser to 1.1.2 (:issue:`MONGOSH-1859`)
```

Comment:
Line needs a terminal period.

Also, note: without the period it seems like `:issue:` eats the closing parenthesis into its title.